test/functional/api/vim_spec.lua

```diff
@@ -294,6 +294,48 @@ describe('API', function()
                                                 |
       ]]}
     end)
+
+    it(':verbose shows correct line numbers', function()
+      -- should work like test_vimscript's Test_function_defined_line()
```

Comment:
Ideally it'd be nice if these tests could be kept in-sync to make sure this behaviour always matches whatever regular :source does.

Maybe I could modify that oldtest to also check anon source output matches? :thinking: